client/src/components/main/Search.js

```diff
@@ -43,57 +48,56 @@ class Search extends Component {
     }
     try {
       const { filter } = this.state;
-      this.props.client.query({
-        query: SEARCH_SHOPS,
-        variables: { filter }
-      }).then(result => {
-        const coffeeShops = result.data.searchShops;
-        let coffees = []
-        coffeeShops.forEach(shop => {
-          coffees = coffees.concat(shop.coffees)
+      this.props.client
```

Comment:
I know this was prob an automatic change, but I would like to keep the original styling and reject this one.